File: 转移台厂商资料/二次开发例程及SDK手册/C# Demo/CSharpDemo/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;

namespace CSharpDemo
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
            MT_API.MT_Init();
        }

        private void btn_Check_Click(object sender, EventArgs e)
        {
            MT_API.MT_Close_UART();
            MT_API.MT_Close_USB();
            if(0!=MT_API.MT_Open_UART(txt_Port.Text))
            {
                MessageBox.Show("串口连接错误");
                return;
            }


            if(0==MT_API.MT_Check())
            {
                MessageBox.Show("ok");
            }
            else
            {
                MessageBox.Show("NO card");
            }
        }

        private void Form1_FormClosed(object sender, FormClosedEventArgs e)
        {
            MT_API.MT_DeInit();
        }

        private void Form1_Load(object sender, EventArgs e)
        {
            
        }

        private void btn_Acc_Click(object sender, EventArgs e)
        {
            Int32 iResult;
            iResult=MT_API.MT_Set_Axis_Acc(0, Convert.ToInt32(txt_Acc.Text));
            if(0==iResult)
            {
                MessageBox.Show("ok");
            }
        }

        private void btn_Dec_Click(object sender, EventArgs e)
        {
            Int32 iResult;
            iResult=MT_API.MT_Set_Axis_Dec(0, Convert.ToInt32(txt_Dec.Text));
            if (0 == iResult)
            {
                MessageBox.Show("ok");
            }
        }

        private void btn_MaxV_Click(object sender, EventArgs e)
        {
            Int32 iResult;
            iResult=MT_API.MT_Set_Axis_Position_V_Max(0, Convert.ToInt32(txt_MaxV.Text));
            if (0 == iResult)
            {
                MessageBox.Show("ok");
            }
        }

        private void btn_Ref_Click(object sender, EventArgs e)
        {
            Int32 iResult;
            MT_API.MT_Set_Axis_Mode_Position(0);
            iResult = MT_API.MT_Set_Axis_Position_P_Target_Rel(0, Convert.ToInt32(txt_Ref.Text));
            if (0 == iResult)
            {
                MessageBox.Show("ok");
            }
        }

        private void btn_Abs_Click(object sender, EventArgs e)
        {
            Int32 iResult;
            MT_API.MT_Set_Axis_Mode_Position(0);
            iResult=MT_API.MT_Set_Axis_Position_P_Target_Abs(0, Convert.ToInt32(txt_Abs.Text));
            if (0 == iResult)
            {
                MessageBox.Show("ok");
            }
        }

        private void btn_Stop_Click(object sender, EventArgs e)
        {
            Int32 iResult;
            iResult=MT_API.MT_Set_Axis_Position_Stop(0);
            if (0 == iResult)
            {
                MessageBox.Show("ok");
            }
        }

        private void btn_Halt_Click(object sender, EventArgs e)
        {
            Int32 iResult;
            iResult=MT_API.MT_Set_Axis_Halt(0);
            if (0 == iResult)
            {
                MessageBox.Show("ok");
            }
        }

        private void btn_Status_Click(object sender, EventArgs e)
        {

        }
    }
}

File: 转移台厂商资料/二次开发例程及SDK手册/VBDemo/VBDemo/Form1.vb
Public Class Form1

    Private Sub Button1_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles Button1.Click

        If (MT_API.R_OK = MT_API.MT_Open_UART("COM1")) Then
            '串口打开成功
            Button1.BackColor = Color.Green
        Else
            Button1.BackColor = Color.Red
        End If
    End Sub

    Private Sub Button2_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles Button2.Click
        '检测指定串口上的板卡连接状态
        If (MT_API.R_OK = MT_API.MT_Check()) Then

            Button2.BackColor = Color.Green
        End If
    End Sub

    Private Sub Button3_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles Button3.Click
        '设置参数
        '加速度
        MT_API.MT_Set_Axis_Acc(0, 1000)
        '减速度
        MT_API.MT_Set_Axis_Dec(0, 1000)
        '速度
        MT_API.MT_Set_Axis_Position_V_Max(0, 2000)
        '工作模式(可以不设置，默认为定位模式)
        MT_API.MT_Set_Axis_Mode_Position(0)
    End Sub

    Private Sub Button4_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles Button4.Click
        '相对当前位置移动10000个单位
        MT_API.MT_Set_Axis_Position_P_Target_Rel(0, 10000)
    End Sub

    Private Sub Form1_Load(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles MyBase.Load
        MT_API.MT_Init()
    End Sub

    Private Sub Form1_FormClosed(ByVal sender As System.Object, ByVal e As System.Windows.Forms.FormClosedEventArgs) Handles MyBase.FormClosed
        MT_API.MT_DeInit()
    End Sub
End Class
